### DISCUSSION TITLE
Add a "feels like" bpm display in song select

### DISCUSSION THREAD

**Discussion by ar1a** [ppy/osu#22774] [Ideas]
Often there are songs that show as 130bpm or something similarly low, but the entire map is 1/6, or 1/8 rhythms. I propose we add a display for a "feels-like" bpm that's normalised to common time. 

Hypothetically, a 130 bpm map all of 1/6ths should have a feels-like bpm of 195bpm (`130 * 6 / 4`). 

 I find maps with this rhythm structure are very common among metal maps. [This is the first that comes to mind](https://osu.ppy.sh/beatmapsets/1762520#osu/3607502)

Another example would be [The Big Black](https://osu.ppy.sh/beatmapsets/41823#osu/131891), while it displays as 360bpm, it's streams are made up of 1/2 notes, and the jumps are made up of 1/1s. this *effectively* plays like most 180bpm maps, so the feels-like bpm would be 180bpm (`360 * 2 / 4`)

**Comment by mk56-spn** [comment]
I'm against this proposal. It sounds good on paper until you remember the amount of " feels like whatever but isn't" edge cases is going to make it a nightmare to create and maintain. Also what's the cut off point? Like do maps with 1/6 and 1/4 get adjusted or not, what's the plan for stuff like  IIL maps ?

**Comment by ar1a** [comment]
I would assume they would go the way of the "mostly x bpm" section as it
currently stands - ignored

Seeing an IIL map thats 63-300 (mostly 137) or something, it would be
probably the same experience with the "feels like" bpm

**Reply by Axilotl17** [reply]
apparently replying on an email doesn't make a reply to mk56-spn's comment, good to know.

**Comment by Axilotl17** [comment]
an alternative suggestion is "note density" which would be `total note count / drain time in seconds` and would be a value like `3 notes / second`. I have seen a proposal like this before for beatmap filtering.
